RELEASE-NOTES.md

```diff
@@ -1,5 +1,53 @@
+* Remove scheduled flow runs when updating/removing schedules by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/12048
+* Move validator logic to API endpoint to avoid issue with legacy client detection by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/12056
+* Update multiple schedules error to say you can update your client by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/12061
+
+### Documentation
+* Improve language and formatting in Profiles and Configuration guide by @discdiver in https://github.com/PrefectHQ/prefect/pull/11996
+* Improves docs formatting consistency and adds some minor content updates by @discdiver in https://github.com/PrefectHQ/prefect/pull/12004
+* Updates formatting for guide: creating-interactive-workflows.md by @discdiver in https://github.com/PrefectHQ/prefect/pull/11991
+* Docs: Add import statement for `wait_for_flow_run` by @urimandujano in https://github.com/PrefectHQ/prefect/pull/11999
+* [docs] add deep dive on overriding job variables by @zzstoatzz in https://github.com/PrefectHQ/prefect/pull/12033
+* Remove extraneous trailing quotation marks in concepts/artifacts docs by @collincchoy in https://github.com/PrefectHQ/prefect/pull/12040
+* Adds links to overriding job variables guide by @discdiver in https://github.com/PrefectHQ/prefect/pull/12043
+
+### Uncategorized
+* disable telemetry for integration tests server by @hamzamogni in https://github.com/PrefectHQ/prefect/pull/12000
+* Switching to the astral-sh pre-commit hook by @chrisguidry in https://github.com/PrefectHQ/prefect/pull/12005
+* [task scheduling] add `TaskServer.execute_task_run` by @zzstoatzz in https://github.com/PrefectHQ/prefect/pull/12002
+* Updates the contact info for our bug bounty program by @chrisguidry in https://github.com/PrefectHQ/prefect/pull/12016
+* Add CLI command to read runs in WQ by @urimandujano in https://github.com/PrefectHQ/prefect/pull/11989
+* [task scheduling] unit test traditional task features by @zzstoatzz in https://github.com/PrefectHQ/prefect/pull/12008
+* Disallow alternate task submission methods by @zzstoatzz in https://github.com/PrefectHQ/prefect/pull/12025
+* Corrects a timing-dependent test for pushing Docker images by @chrisguidry in https://github.com/PrefectHQ/prefect/pull/12038
+* Chore: bump ui dependencies by @brandonreid in https://github.com/PrefectHQ/prefect/pull/12039
+* Add experimental flag for enhanced scheduling UI by @bunchesofdonald in https://github.com/PrefectHQ/prefect/pull/12047
+* Adding a more readable error in case Docker not running  by @eladm26 in https://github.com/PrefectHQ/prefect/pull/12045
+* [task scheduling] send up client id if cloud by @zzstoatzz in https://github.com/PrefectHQ/prefect/pull/12049
+* Revert "Run tasks on the main thread (#11930)" by @chrisguidry in https://github.com/PrefectHQ/prefect/pull/12054
```

Comment:
Should we make this a highlight since it's a significant reversion?

Reply:
I have a weakly held preference for just including it in the fixes section